scripts/jenkins/cloud/ansible/roles/input_model_generator/templates/barclamps/nova.yml.j2

```diff
@@ -23,6 +23,10 @@
       use_migration: true
       kvm:
         ksm_enabled: true
+      compute:
+        kvm-x86_64:
+          cpu_mode: custom
+          cpu_model: kvm64
```

Comment:
Shouldn't the `cpu_model` be set to Westmere? (see: https://github.com/SUSE-Cloud/automation/commit/449057a37cee60b94afca266441e0fcbea1d695f)

Also, I think those overwrites should be used only for virtual deployments (on ECP).

Reply:
Fixed